tck/src/main/java/ee/jakarta/tck/data/standalone/entity/EntityTests.java

```diff
@@ -704,51 +703,53 @@ public void testFindPage() {
     public void testFirstCursoredPageOf8AndNextPages() {
         // The query for this test returns 1-15,25-32 in the following order:
 
-        // 25, 26, 27, 28, 29, 30, 31, 32 square root rounds down to 4
-        // 9, 10, 11, 12, 13, 14, 15 square root rounds down to 3
-        // 4, 5, 6, 7, 8 square root rounds down to 2
-        // 1, 2, 3 square root rounds down to 1
+        // 32 requires 6 bits
+        // 25, 26, 27, 28, 29, 30, 31 requires 5 bits
+        // 8, 9, 10, 11, 12, 13, 14, 15 requires 4 bits
+        // 4, 5, 6, 7, 8 requires 3 bits
+        // 2, 3 requires 2 bits
+        // 1 requires 1 bit
```

Comment:
This test case was trying to assert ordering by square root descending but it actually asks for sorting by number of bits required descending.